library/spdm_crypt_lib/libspdm_crypt_crypt.c

```diff
@@ -4255,15 +4243,23 @@ bool libspdm_verify_leaf_cert_eku_spdm_OID(const uint8_t *cert, size_t cert_size
     size_t index;
     size_t len;
 
+    /* SPDM defined OID */
+    #define OID_SPDM_EXTENSION {0x2B, 0x06, 0x01, 0x04, 0x01, 0x83, 0x1C, 0x82, 0x12, 0x06}
```

Comment:
@jyao1 Should some of these OIDs live in `industry_standard/spdm.h` since they're defined in the specification?

Reply:
ok. Agree. The spec defined data can be in spdm.h